crates/jstzd/src/lib.rs

```diff
@@ -1,8 +1,67 @@
 pub mod docker;
 pub mod task;
+use octez::r#async::client::OctezClient;
+use std::path::Path;
+use tokio::io::AsyncWriteExt;
+
+pub const EXCHANGER_ADDRESS: &str = "KT1F3MuqvT9Yz57TgCS3EkDcKNZe9HpiavUJ";
+pub const JSTZ_ROLLUP_ADDRESS: &str = "sr1PuFMgaRUN12rKQ3J2ae5psNtwCxPNmGNK";
+pub const JSTZ_NATIVE_BRIDGE_ADDRESS: &str = "KT1GFiPkkTjd14oHe6MrBPiRh5djzRkVWcni";
 
 /// The `main` function for running jstzd
 pub async fn main() -> anyhow::Result<()> {
     println!("Hello, world!");
     Ok(())
 }
+
+// this requires a running octez node with any activated protocol version
+pub async fn generate_bootstrap_contract_files(
```

Comment:
i'm thinking of the use cases here and was wondering if we have to expose a function that generates the contract files. we probably need something like `generate_l1_params_files` that will build the rollup installer kernel, preimages and the contracts but do we ever need the user to generate the contract files like this?

Reply:
We don't need to expose this. I placed it in `lib.rs` simply because it contains some lines related to jstzd, specifically the contract init data part, and thus I don't think putting it in `contract_test` is that appropriate. It's `pub` just because it needs to be imported by the test.

I'll move it to `contract_test` to avoid exposing this unnecessarily.